index.html

```diff
@@ -185,5 +185,18 @@ <h4>Skills</h4>
          </div>
       </section>
    </main>
+   <footer id="footer">
+      <section id="footer_form">
+         <h2>Contact me</h2>
+         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+         <form action="https://formspree.io/f/mpzeaaag" method="post" class="contact_form">
+            <input class="form_item" type="text" name="name" placeholder="Yeremias" maxlength="30" required>
+            <input class="form_item" type="email" name="email" placeholder="natajayanj14@gmail.com" required>
```

Comment:
- [OPTIONAL] Your form looks great, but you used some specific placeholder text for your input. It would be better with a more general placeholder like 'Name' and 'Email' for example.

Reply:
Oh. Thanks, @leonardodiasb for this feedback. I will take note of this